### DISCUSSION TITLE
스터디 2기 1주차 모임

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#296] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. Sam입니다👋

오늘 시작부터 우여곡절이 조금 있었는데 모두 끝까지 OT와 온보딩에 참여해주셔서 감사드립니다ㅎㅎ

두 번째 모임부터는 오늘 달레님이 말씀하신대로 각 문제를 풀이하고 피드백 및 논의를 하는 시간을 가질 예정입니다! 

문제당 한 분씩 풀이를 할 예정이고, 신청은 선착순으로 각 주차에 있는 아직 풀이 희망자가 없는 문제 중 원하시는 문제와 디스코드 명을  one-bit-integer / Sam 과 같은 형식으로 댓글로 남겨주시면 됩니다!ㅎㅎ

다음 모임도 이전과 마찬가지로 아래 링크로 RSVP 부탁드리겠습니다!(https://discord.gg/SfxHQvHq?event=1272061430249553970)

**Comment by YeonguChoe** [comment]
Top k frequent element/ 영우

**Comment by DaleSeo** [comment]
코드 리뷰의 좋은 예: https://github.com/DaleStudy/leetcode-study/pull/302, https://github.com/DaleStudy/leetcode-study/pull/299

**Comment by obzva** [comment]
Kth smallest element in a BST / Flynn

**Comment by Sunjae95** [comment]
Palindromic Substrings / 선재

**Comment by TonyKim9401** [comment]
Number of 1 Bits / 김토니

**Comment by DaleSeo** [comment]
기타 논의 사항
- 언어 Tag 활용
- 잦은 nitpick
  - 파일 마지막 line break 
  - 프로젝트 미설정 (Status, Iteration 대신에 Week로 용어 통일)
- GitHub와 Discord 유저네임 혼란
  - `닉네임(깃헙username)`로 통일하기로 결정

**Comment by HC-kang** [comment]
contains-duplicate / 강희찬
